inc/namespace.php

```diff
@@ -108,8 +108,12 @@ function remove_2fa_dummy_provider( array $providers ) : array {
  * @return bool
  */
 function override_two_factor_universally_forced( bool $is_forced ) : bool {
+	if ( maybe_skip_two_factor_forcing_for_local_env() ) {
+		return false;
+	}
+
 	$config = Altis\get_config()['modules']['security']['2-factor-authentication'];
-	if ( is_array( $config ) && ( ! empty( $config['required'] ) || is_bool( $config['required'] ) ) ) {
+	if ( is_array( $config ) && ( ! empty( $config['required'] ) && is_bool( $config['required'] ) ) ) {
```

Comment:
This breaks the functionality for setting `$config['required']` to `false`. The function return type hint will cast the value to a boolean even if it's an array.
```suggestion
	if ( is_array( $config ) && ( ! empty( $config['required'] ) || is_bool( $config['required'] ) ) ) {
```

Reply:
Hmmmm.. it broke the feature when `required` was an array, the function returned the array in that case, against its return type hint, resulting in a fatal.

I think we should use `isset` in this case maybe ?

```suggestion
	if ( is_array( $config ) && ( isset( $config['required'] ) && is_bool( $config['required'] ) ) ) {
```

Reply:
Ah ok, I tried a test on the command line - you're right it does fatal so we need to cast the return value to a bool on the line below, e.g. `return (bool) $config['required'];`

Reply:
I went with `isset( $config['required'] && is_bool( $config['required'] )` as it handled any boolean value, true or false, and ignores anything that's not a boolean.